microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/EnableJwtSubsystemSetupTask.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.eap.qe.microprofile.jwt;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.MicroProfileServerSetupTask;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+/**
+ * A way enable/disable MP-JWT subsystem when not enabled
+ */
+public class EnableJwtSubsystemSetupTask implements MicroProfileServerSetupTask {
+
+    private static final Address MP_JWT_EXTENSION_ADDRESS = Address.extension("org.wildfly.extension.microprofile.jwt");
+    private static final Address MP_JWT_ADDRESS = Address.subsystem("microprofile-jwt-smallrye");
+
+    private boolean wasExtensionAdded;
+    private boolean wasSubsystemAdded;
+
+    @Override
+    public void setup() throws IOException, OperationException, TimeoutException, InterruptedException,
+            ConfigurationException {
+        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            final Operations ops = new Operations(client);
+            /*
+             * if (!ops.exists(MP_JWT_EXTENSION_ADDRESS)) {
```

Comment:
@honza-kasik I thought we want to add extension if it is not there by default? Also `FaultToleranceServerSetup` do so.

Reply:
Thanks, this is already addressed in commit I forgot to push, so I'll have to restart the job.